EiDiA_frontend/src/components/record/filetable/Element.js

```diff
@@ -0,0 +1,99 @@
+"use strict";
+
+import React from 'react';
+import styled from "styled-components";
+import {ElementSymbol} from "./ElementSymbol";
+import {ElementActions} from "./ElementActions";
+import {DragTypes} from "../../../assets/Constants"
+import {DragSource} from "react-dnd";
+import Grid from "@material-ui/core/Grid";
+
+
+const Name = styled.div`
+    
+    padding-left: ${props => props.padding + "%"};
+`;
+
+
+
+const itemSource = {
+    beginDrag() {
+        return {};
+    },
+    endDrag(props, monitor, component) {
+        if (!monitor.getDropResult()) return
+        let wrapperProps = monitor.getDropResult().component.props;
+        component.props.handleDrop(wrapperProps.id);
+
+    }
+}
+
+function collectDrag(connect, monitor) {
+    return {
+        connectDragSource: connect.dragSource(),
+        isDragging: monitor.isDragging()
+    };
+}
+
+class Element extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            symbolArray: [],
+            padding: this.props.level * 5,
+            width: 40 - this.props.level * 2,
+            elementData: this.props.elementData
+        };
+    }
+
+
+    render() {
+        const {isDragging} = this.props;
+
+        const toRender = (
+            <div>
+                {!isDragging &&
+                //TODO: cursor doesnt work yet
+                <Grid container spacing={2} style={{cursor: (this.props.type === 'FOLDER') && 'pointer'}}
+                      onClick={this.props.activeToggle}>
+                    <Grid item xs={12} sm={4}>
+                        <Name padding={this.state.padding}>
+                            <ElementSymbol type={this.state.elementData.type}
+                                           active={this.state.elementData.active}/>
+                            {this.state.elementData.name}
```

Comment:
Abstand Symbol <-> Titel
```suggestion
                            &nbsp;&nbsp;{this.state.elementData.name}
```